test/functional/feature_backwards_compatibility.py

```diff
@@ -136,24 +137,24 @@ def run_test(self):
         wallet = node_v18.get_wallet_rpc("w1_v18")
         info = wallet.getwalletinfo()
         assert info['private_keys_enabled']
-        assert info['keypoolsize'] > 0
+        assert_greater_than(info['keypoolsize'], 0)
 
         # w2: wallet with private keys disabled, created on master: update this
         #     test when default wallets private keys disabled can no longer be
         #     opened by older versions.
         node_master.createwallet(wallet_name="w2", disable_private_keys=True)
         wallet = node_master.get_wallet_rpc("w2")
         info = wallet.getwalletinfo()
-        assert info['private_keys_enabled'] == False
-        assert info['keypoolsize'] == 0
+        assert_equal(info['private_keys_enabled'], False)
```

Comment:
Since you added `assert_true`, should we also have `assert_false`?

Reply:
Same https://github.com/bitcoin/bitcoin/pull/25732#discussion_r934898796

Reply:
I would like use some helper function because IMHO this make the code more readble but this is up to up guys

Reply:
On second thought, I think `assert_equal` here works well, since the previous code was indeed just comparing equality and not doing a boolean comparison. Resolved.